configure.ac

```diff
@@ -862,5 +878,6 @@ Build options:
   XFixes client disconnect      $have_xfixes
   Run test cases                $enable_tests
   Install tests                 $enable_install_tests
+  Install systemd service       $enable_systemd
```

Comment:
I like to put this line between "XFixes client disconnect" and "Run test cases" lines, i.e, before test options.

Reply:
Sure, did that.